### ISSUE TITLE
Pagination on results

### ISSUE THREAD

**Issue by EladioGaldamez** [Aratramba/sanity-plugin-media-library#38]
## Problem
We do have +2K media files and the media library takes up to 20 - 30 seconds to start showing the images.

## Possible Solution
Pagination or infinite scrolling of media files & assets would be a nice way to solve it. The default Sanity media selector already includes a pagination feature.

**Comment by Aratramba** [comment]
Yes, pagination would be a good solution for that. I don't have much time to work on that right now, but I'm happy to accept a PR that addresses this issue.
